list_to_email_converter.py

```diff
@@ -0,0 +1,234 @@
+import os
+from urllib.parse import quote
+import sys
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("path",type=str, help="the folderpath like: ./example/more/pytr")
+parser.add_argument("-s","--seek", help="seeks an existing entry", action="store_true")
```

Comment:
This approach will also help to avoid the situation where you have specified `-s` but a full path (which will result in an error at the moment).